mobileApp/src/components/board/square.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {FC} from 'react';
+import {useSelector, useDispatch} from 'react-redux';
+import {clickOnSquare} from '../game';
+import {IInitState} from '../../redux/reducers';
+import {
+  Disabled,
+  Index,
+  SquaresInHistory,
+  StepNumber,
+  XIsNext,
+} from '../../types';
+import {StyleSheet, TouchableHighlight, View, Text} from 'react-native';
+
+export type SquareProps = {index: Index};
+
+export const Square: FC<SquareProps> = (prop) => {
+  const {index} = prop;
+  const history = useSelector<IInitState, SquaresInHistory>(
```

Comment:
Please, create folder `selectors` into redux folder for saving all selectors with its types. After you can use it from this place everywhere. You have some duplicated selectors for now.